### PR TITLE
[15.0] [MIG] `sale_order_report_product_image` : Migration to 15.0

### PR DESCRIPTION
**Pull request by ivantodorovich**

### CODE REVIEW

File: sale_order_report_product_image/i18n/sale_order_report_product_image.pot [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* sale_order_report_product_image
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```
Comment (cyrilmanuel) [inline sale_order_report_product_image/i18n/sale_order_report_product_image.pot:7]: "Project-Id-Version: Odoo Server 15.0\n" shall be update ?

**Review by cyrilmanuel — APPROVED** [review]

File: sale_order_report_product_image/i18n/sale_order_report_product_image.pot (hunk 1 above)
Comment (StephaneMangin) [reply sale_order_report_product_image/i18n/sale_order_report_product_image.pot:7]: Yes but translations are not manage in the github repo but by an external tool named Weblate here : https://translation.odoo-community.org/

**Review by StephaneMangin — APPROVED** [review]

File: sale_order_report_product_image/i18n/sale_order_report_product_image.pot (hunk 1 above)
Comment (cyrilmanuel) [reply sale_order_report_product_image/i18n/sale_order_report_product_image.pot:7]: Okey, thanks for the explanation :+1:

**Pull request merged**